src/io.jl

```diff
@@ -46,19 +44,19 @@ function create_parameters_and_sets_from_file(input_folder::AbstractString)
     p_unit_capacity   = Dict{String,Float64}()
     p_init_capacity   = Dict{String,Float64}()
     for row in eachrow(nodes_data_df)
-        if row.node_name in Ap
-            p_variable_cost[row.node_name] = row.variable_cost
-            p_investment_cost[row.node_name] = row.investment_cost
-            p_unit_capacity[row.node_name] = row.asset_capacity
-            p_init_capacity[row.node_name] = row.initial_capacity
+        if row.name in Ap
```

Comment:
Is this the proper way of checking an item is in a set? Seems like the set should be by ID and not Name.

Reply:
Nvm, this is addressed in another issue.